source/upgrade-guide/compatibility-matrix/index.rst

```diff
@@ -29,33 +29,10 @@ The Wazuh manager is also compatible with OSSEC agents but not all capabilities
 Elastic and the Wazuh Kibana plugin
 -----------------------------------
 
-The following Elastic Stack and Open Distro for Elasticsearch versions are compatible with the Wazuh manager |WAZUH_CURRENT| using the Wazuh Kibana plugin:
+Elastic Stack and Open Distro for Elasticsearch are no longer compatible with the Wazuh manager since v4.6.0.
```

Comment:
```suggestion
Since v4.6.0, we no longer support Elastic Stack nor Open Distro for Elasticsearch. Check :doc:`/integrations-guide/index` for information on how to integrate Wazuh with them.
```